packaging/RPMS/Fedora/rabbitmq-server.spec

```diff
@@ -188,7 +190,9 @@ fi
 %attr(0755, rabbitmq, rabbitmq) %dir %{_localstatedir}/lib/rabbitmq
 %attr(0750, rabbitmq, rabbitmq) %dir %{_localstatedir}/lib/rabbitmq/mnesia
 %attr(0755, rabbitmq, rabbitmq) %dir %{_localstatedir}/log/rabbitmq
-%dir %{_sysconfdir}/rabbitmq
+%attr(2755, -, -) %dir %{_sysconfdir}/rabbitmq
```

Comment:
Yes, that's probably fine. I don't know RPM packages very well, so let's trust documentation :)